Makefile

```diff
@@ -394,11 +394,13 @@ static: skydive.clean govendor genlocalfiles
 contribs.clean:
 	$(MAKE) -C contrib/snort clean
 	$(MAKE) -C contrib/objectstore clean
+	$(MAKE) -C contrib/ebpf-check clean
 
 .PHONY: contribs
 contribs:
 	$(MAKE) -C contrib/snort
 	$(MAKE) -C contrib/objectstore
+	$(MAKE) -C contrib/ebpf-check
```

Comment:
It is a bit unusual to have "functional test" running while just building. Moreover we have to wait 2sec because of this each time we build. The test itself is not enough robust there is no guaranty that we will have packet within 2sec. I really think we should remove this. make test or test.functional does already the job

Reply:
my attention was to build it only ... fixed

Reply:
I don't see the point to build it. If it is a dev tool it should not be there, we all have our own dev tools. Functional tests make sense. Building a binary not being used I'm not sure.